parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueWriters.java

```diff
@@ -71,6 +71,10 @@ public static UnboxedWriter<Short> shorts(ColumnDescriptor desc) {
     return new ShortWriter(desc);
   }
 
+  public static <T> ParquetValueWriter<T> unboxed(ColumnDescriptor desc) {
```

Comment:
Needed to expose this to get `float` and `double` working. Those writers currently require a metrics builder that expects a non-null field ID.